homeassistant/components/humidifier/device_action.py

```diff
@@ -0,0 +1,112 @@
+"""Provides device automations for Humidifier."""
```

Comment:
Let's not include this in the initial PR. It should be as simple as possible to make it easier to review.

Reply:
done